### PR TITLE
update adding IDP section

### PR DESCRIPTION
**Pull request by lamek**
Edits throughout and updating the "Add and IDP" section to call out where instructions do not need to be GitHub specific.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,13 +38,7 @@ Unless your just testing out MOA, we recommend using a dedicated AWS account to
...
 If you are using AWS organizations and you need to have a Service Control Policy (SCP) applied to the AWS account you plan to use, see the [Red Hat Requirements for Customer Cloud Subscriptions](https://www.openshift.com/dedicated/ccs#scp) for details on the minimum required SCP.
 
-As part of the cluster creation process, `moactl` will perform the following actions:
-
-- Create an osdCcsAdmin IAM user:
-  - This user will have Programmatic access enabled.
-  - This user will have the AdministratorAccess policy attached to it.
-  - This user will leverage the IAM credentials you provide when configuring the AWS cli in the next section.
-
+As part of the cluster creation process, `moactl` will create an osdCcsAdmin IAM user. This user will have Programmatic access enabled and have the AdministratorAccess policy attached to it. The user will leverage the IAM credentials you provide when configuring the AWS cli in the next section.
```
Comment (jharrington22) [inline README.md:41]: Once the user is created its no longer leveraging the provided credentials. The last sentence (to me :smile: ) could be changed to something like "The provided AWS credentials in the next section will be used to create this user."

File: README.md [hunk 2]
```diff
@@ -167,9 +160,9 @@ I: AWS quota ok
 
 If you need to increase your quota, navigate to your [AWS console](https://aws.amazon.com/console/), and request a quota increase for the service that failed.
 
-Once both the permissions and quota checks pass, proceed to initializing your AWS account.
+Once both the permissions and quota checks pass, proceed to pareparing your AWS account for cluster installation.
```
Comment (jharrington22) [inline README.md:163]: Typo "Preparing"

**Comment by vkareh** [comment]
@lamek can you squash these commits?

**Comment by lamek** [comment]
@vkareh all set. Thanks.

**Comment by vkareh** [comment]
Thanks! /lgtm

**Pull request merged**